frontend/src/components/jobs/filters/filter-section.tsx

```diff
@@ -46,6 +46,14 @@ export default function FilterSection({ _totalJobs }: FilterSectionProps) {
           filterKey="jobTypes"
           options={[...JOB_TYPES]}
         />
+        <Button
+          className=" border-5 rounded-3xl"
+          size="compact-md"
+          variant="transparent"
+          onClick={() => clearFilters()}
+        >
+          <a className="font-light">clear all</a>
```

Comment:
Should clear all be capitalized? (Clear all)

Reply:
Looks better in all small imo

But if you want it capitalized

```suggestion
          <a className="font-light">Clear all</a>
```